newsletter-bundle/src/Cron/PurgeSubscriptionsCron.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\NewsletterBundle\Cron;
+
+use Contao\CoreBundle\Framework\ContaoFramework;
+use Contao\CoreBundle\ServiceAnnotation\CronJob;
+use Contao\NewsletterRecipientsModel;
+use Psr\Log\LoggerInterface;
+
+/**
+ * @CronJob("daily")
+ */
+class PurgeSubscriptionsCron
+{
+    public function __construct(private ContaoFramework $framework, private LoggerInterface|null $logger)
+    {
+    }
+
+    public function __invoke(): void
+    {
+        $this->framework->initialize();
+
+        $subscriptions = $this->framework->getAdapter(NewsletterRecipientsModel::class)->findExpiredSubscriptions();
```

Comment:
I was confused by this variable name. They are not subscriptions but recipients, maybe you want to change that 🙃

Reply:
I used that variable name because of the method name.